test/tests/functional/pbs_ralter.py

```diff
@@ -390,17 +390,16 @@ def alter_a_reservation(self, r, start, end, shift,
                 msg = "Resv;" + r + ";Reservation alter confirmed"
             else:
                 msg = "Resv;" + r + ";Reservation alter denied"
-            attempts = 1
             interval = 0.5
-            max_attempts = 30
-            while attempts <= max_attempts:
+            max_attempts = 20
+            for attempts in range(1, max_attempts + 1):
                 lines = self.server.log_match(msg, n='ALL', allmatch=True,
-                                              interval=1, max_attempts=30)
+                                              max_attempts=5)
                 info_msg = "log_match: searching " + \
                     str(sequence) + " sequence of message: " + \
                     msg + ": Got: " + str(len(lines))
                 if len(lines) == sequence:
-                    self.assertEqual(len(lines), sequence, info_msg)
+                    self.logger.info(info_msg)
```

Comment:
This logger line is in both sides of the if statement.  Move it just above the if statement.

Reply:
Modified.